authn/config/config.go

```diff
@@ -5,19 +5,27 @@ import (
 
 	"github.com/aserto-dev/go-utils/authn/apikey"
 	"github.com/aserto-dev/go-utils/authn/auth0"
+	"github.com/aserto-dev/go-utils/grpcclient"
 )
 
 type Config struct {
-	APIKeys apikey.Config `json:"api_keys"`
-	Auth0   auth0.Config  `json:"auth0"`
-	Options CallOptions   `json:"options"`
+	APIKeys    apikey.Config     `json:"api_keys"`
+	Auth0      auth0.Config      `json:"auth0"`
+	STSService grpcclient.Config `json:"sts"`
+	Options    CallOptions       `json:"options"`
 }
 
 type CallOptions struct {
 	Default   Options           `json:"default"`
+	Cache     CacheOptions      `json:"cache"`
 	Overrides []OptionOverrides `json:"overrides"`
 }
 
+type CacheOptions struct {
```

Comment:
These cache options seem to be specific to the machine account authenticator. Perhaps they should be defined as part of STSService? Maybe there should be an `STSConfig` struct that contains both `CacheOptions` and `grpcclient.Config`?

CallOptions were intended to be options that can be overridden for individual API paths.